studio/components/interfaces/BillingV2/Usage/Usage.constants.tsx

```diff
@@ -134,33 +136,80 @@ export const USAGE_CATEGORIES: CategoryMeta[] = [
         links: [
           {
             name: 'Documentation',
             url: 'https://supabase.com/docs/guides/platform/database-size',
           },
         ],
         chartDescription: 'The data refreshes every 24 hours.',
-        additionalInfo: (subscription?: ProjectSubscriptionResponse, usage?: ProjectUsageData) =>
-          subscription?.plan?.id !== 'free' ? (
-            <div className="flex items-center justify-between">
-              <div className="flex items-center space-x-2">
-                <p className="text-sm">Disk size:</p>
-                <p className="text-sm">{usage?.disk_volume_size_gb} GB</p>
-                <Badge color="green" size="small">
-                  Auto-scaling
-                </Badge>
+        additionalInfo: (
+          ref: string,
+          subscription?: ProjectSubscriptionResponse,
+          usage?: ProjectUsageData
+        ) => {
+          const usageMeta = usage?.['db_size'] ?? undefined
+          const usageRatio =
+            typeof usageMeta !== 'number' ? (usageMeta?.usage ?? 0) / (usageMeta?.limit ?? 0) : 0
+          const hasLimit = usageMeta && usageMeta.limit > 0
+
+          const isApproachingLimit = hasLimit && usageRatio >= USAGE_APPROACHING_THRESHOLD
+          const isExceededLimit = hasLimit && usageRatio >= 1
+          console.log({ usageMeta, isApproachingLimit, isExceededLimit })
+
+          return subscription?.plan?.id !== 'free' ? (
+            <div>
+              <div className="flex items-center justify-between">
+                <div className="flex items-center space-x-2">
+                  <p className="text-sm">Disk size:</p>
+                  <p className="text-sm">{usage?.disk_volume_size_gb} GB</p>
+                  <Badge color="green" size="small">
+                    Auto-scaling
+                  </Badge>
+                </div>
+                <Link href="https://supabase.com/docs/guides/platform/database-usage#disk-management">
+                  <a>
+                    <Button size="tiny" type="default" icon={<IconExternalLink size={14} />}>
+                      What is disk size?
+                    </Button>
+                  </a>
+                </Link>
               </div>
-              <Link href="https://supabase.com/docs/guides/platform/database-usage#disk-management">
-                <a>
-                  <Button size="tiny" type="default" icon={<IconExternalLink size={14} />}>
-                    What is disk size?
-                  </Button>
-                </a>
-              </Link>
+              <p className="text-sm text-scale-1000 mt-3">
+                Your disk storage expands automatically when the database reaches 90% of the disk
```

Comment:
Moved these around as the read-only piece seems like the most important(?): 


If you reach 95% of the disk space, your project will enter read-only mode. 
We will automatically expand your disk by 50% when the database reaches 90% of the disk size. 
Auto-scaling can only take place once every 6 hours. If you know you will need more than 50%, you can manually resize the disk to be what you need.